src/community/mod.rs

```diff
@@ -172,3 +172,21 @@ impl Community {
         }
     }
 }
+
+pub const DEVHUB_COMMUNITY_FACTORY: &str = "community.devhub.near";
```

Comment:
Could we verify that the community handle is all lowercase, e.g. don't allow nearHacks.community.devhub.near, only nearhacks.community.devhub.near

Reply:
Sure. Yeah you're right, near account cannot have uppercase. There are two existing community violate this rule, I'll manually rename their handle